app/views/content_items/html_publication.html.erb

```diff
@@ -13,12 +13,17 @@
 </div>
 
 <header class="publication-header" id="contents">
-  <div class="html-publication-title">
+  <% @context = capture do %>
     <% if @content_item.format_sub_type %>
-      <p class="context"><%= I18n.t("content_item.schema_name.#{@content_item.format_sub_type}", count: 1) %></p>
+      <%= I18n.t("content_item.schema_name.#{@content_item.format_sub_type}", count: 1) %>
     <% end %>
-    <h1><%= @content_item.title %></h1>
-  </div>
+  <% end %>
+
+  <%= render partial: 'govuk_component/title', locals: {
+    title: @content_item.title,
+    context: @context,
```

Comment:
I think could be simplified to:

```
context: I18n.t("content_item.schema_name.#{@content_item.format_sub_type}", count: 1)
```

Reply:
@andysellick’s change (https://github.com/alphagov/government-frontend/pull/474) means `format_sub_type` will always be included.